server/command/main.go

```diff
@@ -54,3 +57,12 @@ func runSubcommand(
 func (s *service) executeDebugClean(params *params) (*model.CommandResponse, error) {
 	return normalOut(params, md.MD("<><> TODO"), nil)
 }
+
+func (s *service) executeDebugLocations(params *params) (*model.CommandResponse, error) {
+	locations, err := s.apps.API.GetLocations(params.commandArgs.UserId, params.commandArgs.ChannelId)
+	if err != nil {
+		return normalOut(params, md.MD("error"), err)
+	}
+	bytes, _ := json.Marshal(locations)
+	return normalOut(params, md.MD(string(bytes)), nil)
```

Comment:
For better formatting
```suggestion
	return normalOut(params, md.JSONBlock(locations), nil)
```